igvc_navigation/config/teb_local_planner_params.yaml

```diff
@@ -33,7 +33,7 @@ TebLocalPlannerROS:
   # GoalTolerance
 
   xy_goal_tolerance: 0.2
-  yaw_goal_tolerance: 1.57
+  yaw_goal_tolerance: 3.14
```

Comment:
As I mentioned in Software Core, this probably doesn't matter. The tolerance is a plus or minus thing so both of these should have the same effect. I'm also still getting curved paths when approaching waypoints in simulation.

Reply:
Maybe we should open an issue to understand what's actually causing this? Doesn't really feel like a priority to me and could be added to the TEB tuning task instead of being an independent issue, but still good to make a note for later.

Reply:
yeah, that is true. Ok, I'll create a bug for it.